add current room sensor to Roborock (#32244)
home-assistant/home-assistant.github.io: source/_integrations/roborock.markdown

### Sensor

> Current room - What room your vacuum is currently in according to the last map update. It gives your vacuums last known location in your current selected map similar to the app.

Reviewer: ```suggestion
Current room—The room your vacuum is currently in according to the last map update. Similar to the app, this gives your vacuum's last known location in your current selected map.
```
